app/controllers/katello/api/v2/host_subscriptions_controller.rb

```diff
@@ -35,6 +37,42 @@ def events
       respond_for_index :collection => collection
     end
 
+    api :POST, "/hosts/subscriptions/register", N_("Simulate a host subscription-manager registration")
```

Comment:
@ehelms are you saying:

a) there is no need for creating a host with subscription information out of using 'subscription-manager register'
or
b) if someone wants to create a host with subscription information they should use the rhsm api?
or 
c) something else?